### PR TITLE
Do not collect duplicated information

### PR DESCRIPTION
**Pull request by Jaanki**
Submariner, k8s, OS and kernel information
are collected as part of `subctl gather`. Don't
ask for them explicitly in bug report.

Attaching summary.html (which includes all necessary information) for reference.
summary.html is created while running `subctl gather`
[summary.html.log](https://github.com/submariner-io/submariner-operator/files/6825233/summary.html.log)

(Renaming it to .log because .html is not a supported extension)

Signed-off-by: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (dfarrell07) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]: Gather collects these from the host, not just containers?

Comment (Jaanki) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: @dfarrell07 Yes. From each node by parsing `Node.Status.NodeInfo`
https://github.com/submariner-io/submariner-operator/blob/devel/pkg/subctl/cmd/gather/summary.go#L154-L167

and the output looks like this (not properly formatted here due to copy paste)

```
Node name | Operating System | Container Runtime Version | Kubelet Version | KubeProxy Version
ip-10-0-129-36.ec2.internal | linux Red Hat Enterprise Linux CoreOS 47.83.202106252242-0 (Ootpa) 4.18.0-240.22.1.el8_3.x86_64 amd64 | cri-o://1.20.3-6.rhaos4.7.git0d0f863.el8 | v1.20.0+87cc9a4 | v1.20.0+87cc9a4
```

**Review by tpantelis — APPROVED** [review]

**Comment by nyechiel** [comment]
@Jaanki can you please handle the same in the rest of the repo? thanks!

**Review by nyechiel — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (dfarrell07) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: Right, so that's the node OS/similar. I understood the Issue Template to be asking for the host OS/similar, like the machine you're running `subctl` from. That's what the directions (`cat /etc/os-release`) will get you.

**Comment by Jaanki** [comment]
> @Jaanki can you please handle the same in the rest of the repo? thanks!

Yes @nyechiel Once this is merged, I will add PRs in other repos

**Comment by dfarrell07** [comment]
> > @Jaanki can you please handle the same in the rest of the repo? thanks!
> 
> Yes @nyechiel Once this is merged, I will add PRs in other repos

I'll assign https://github.com/submariner-io/submariner-website/issues/101 to you then if that's okay, as it track exactly that TODO.

File: .github/ISSUE_TEMPLATE/bug-report.md
Comment (Jaanki) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: Sure @dfarrell07 Also if you can vote on this issue and merge it please.

Comment (dfarrell07) [reply .github/ISSUE_TEMPLATE/bug-report.md:31]: I think we merged this without addressing my -1.

We shouldn't dismiss reviews unless the feedback is clearly fixed.

**Pull request merged**
